config.go

```diff
@@ -228,6 +230,35 @@ type Config struct {
 	// RequireNodeNames controls if the name of a node is required when sending
 	// a message to that node.
 	RequireNodeNames bool
+	// If nil, allow any connection (default), otherwise specify all networks
+	// allowed to connect (you must specify IPv6/IPv4 separately)
+	// Using [] will block all connections.
+	CidrsAllowed []net.IPNet
```

Comment:
trivial: Just realized as this is an initialism, go conventions say it should be `CIDRsAllowed` or `AllowedCIDRs` like `net.ParseCIDR` and `net.CIDRMask`.

Reply:
DONE